IntegrationTests/tests_01_http/test_03_post_random_bytes.sh

```diff
@@ -13,11 +13,12 @@
 ##
 ##===----------------------------------------------------------------------===##
 
+# shellcheck source=IntegrationTests/tests_01_http/defines.sh
 source defines.sh
 
 token=$(create_token)
 start_server "$token"
-dd if=/dev/urandom of="$tmp/random.bytes" bs=$((64*1024)) count=1
+dd if=/dev/urandom of="${tmp:?}/random.bytes" bs=$((64*1024)) count=1
```

Comment:
We're seeing this pattern a lot in this PR. Are all these things coming from `defines.sh` presumably? 

If not, why are we not explicitly setting it somewhere?

Reply:
Those don't come from the `defines.sh` but rather `run-tests.sh` is setting the tmp dir and then all other scripts are executed with that from what I can tell